src/engraving/rendering/score/parenthesislayout.cpp

```diff
@@ -466,3 +453,29 @@ void ParenthesisLayout::setDefaultValues(Parenthesis* item, Parenthesis::LayoutD
     const double PADDING = spatium * 0.2;
     ldata->setPosX(item->direction() == DirectionH::RIGHT ? bbox.right() + PADDING : bbox.left() - PADDING);
 }
+
+Shape ParenthesisLayout::getParentShape(const EngravingItem* parent)
+{
+    Shape parentShape = parent->shape();
+
+    bool isNote = parent->isNote();
+
+    parentShape.remove_if([isNote](ShapeElement& s) {
+        return !s.item() || s.item()->isParenthesis()
+               || (s.item()->isLaissezVibSegment() && isNote);
+    });
```

Comment:
Just making a note for myself: I'd like to implement the equivalent of SkylineLine::getFilteredCopy for the Shape class. Cause removing elements from a vector is an expensive operation and I think just filling a second vector with the elements that match the required condition is a lot faster. Unless we have a reason modify the original Shape objects obviously, which in most cases we don't.